drivers/net/bnxt/bnxt.h

```diff
@@ -58,6 +58,7 @@ struct bnxt_pf_info {
 #define BNXT_FIRST_PF_FID	1
 #define BNXT_MAX_VFS(bp)	(bp->pf.max_vfs)
 #define BNXT_FIRST_VF_FID	128
+#define BNXT_MAX_VF_NUM		128
```

Comment:
I'm not sure this should be a macro since it can very by device.  Also, it doesn't seem to be used anywhere.